plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+option(ENABLE_QSV11 "Build Intel QSV11 hardware encoder." TRUE)
+
+if(NOT ENABLE_QSV11)
+	message(STATUS "NOT building Intel QSV Encoder.")
```

Comment:
Just noticed that the two strings here are asymmetrical:   "Intel QSV11 hardware encoder" vs. "Intel QSV Encoder".

I'd prefer to pick one phrase and use it in both.

Reply:
Done